backend/endpoints/sync.py

```diff
@@ -94,7 +94,7 @@ async def dm_install(payload: InstallPayload):
         400: {"detail": "Object id is not valid"},
     },
 )
-async def get_sidebar(org_id: str, member_id: str, room_type: RoomType):
+async def get_sidebar(org: str, user: str):
```

Comment:
@Bwise1 
Since the repository handles for both DM and Channels, without the roomtype as a query parameter it's going to be difficult to include what roomtype data you're requesting for to be rendered on the sidebar

Reply:
We can ask zuri main to provide `room_type`, but that would not be generalizable to other plugins. A better approach is to provide sidebar info for both at once. However, this needs to be discussed with respect to the payload structure for sidebar.